### PR TITLE
ts_genericcamera snake_case renaming

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

**Review by wvreeven — CHANGES_REQUESTED** [review]
This is a good start. But many methonds with camelCase names remain, like `_assertHandle`, `stopExposure` and many more. The tricky part of this ticket is to replace all camelCase variable and method names with snake_case but leave the ones that refer to native variables and methods in place.

File: python/lsst/ts/genericcamera/driver/zwocamera.py [hunk 1]
```diff
@@ -735,183 +735,183 @@ def getCameraProperty(self, index):
         # *pASICameraInfo, int iCameraIndex);
         cameraProperty = ASICameraInfoCtypes()
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/zwocamera.py:736]: I think that `cameraProperty` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/zwocamera.py [hunk 2]
```diff
@@ -735,183 +735,183 @@ def getCameraProperty(self, index):
...
         result = self.lib.ASIGetCameraProperty(cameraProperty, index)
-        return self._toResultEnum(result), ASICameraInfo(cameraProperty)
+        return self._to_result_enum(result), ASICameraInfo(cameraProperty)
 
     def getCameraPropertyByID(self, id):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/zwocamera.py:740]: I think that `getCameraPropertyByID` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/zwocamera.py [hunk 3]
```diff
@@ -735,183 +735,183 @@ def getCameraProperty(self, index):
...
         # ASICAMERA_API ASI_ERROR_CODE ASIGetCameraPropertyByID(int iCameraID,
         # ASI_CAMERA_INFO *pASICameraInfo);
         cameraProperty = ASICameraInfoCtypes()
         result = self.lib.ASIGetCameraPropertyByID(id, cameraProperty)
-        return self._toResultEnum(result), ASICameraInfo(cameraProperty)
+        return self._to_result_enum(result), ASICameraInfo(cameraProperty)
 
     def openCamera(self, id):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/zwocamera.py:747]: I think that `openCamera` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/zwocamera.py [hunk 4]
```diff
@@ -735,183 +735,183 @@ def getCameraProperty(self, index):
...
         # ASICAMERA_API  ASI_ERROR_CODE ASIOpenCamera(int iCameraID);
         result = self.lib.ASIOpenCamera(id)
-        return self._toResultEnum(result)
+        return self._to_result_enum(result)
 
     def initCamera(self, id):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/zwocamera.py:752]: I think that `initCamera` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/zwocamera.py [hunk 5]
```diff
@@ -735,183 +735,183 @@ def getCameraProperty(self, index):
...
         # ASICAMERA_API  ASI_ERROR_CODE ASIInitCamera(int iCameraID);
         result = self.lib.ASIInitCamera(id)
-        return self._toResultEnum(result)
+        return self._to_result_enum(result)
 
     def closeCamera(self, id):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/zwocamera.py:757]: I think that `closeCamera` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/andorcamera.py [hunk 6]
```diff
@@ -1059,7 +1059,7 @@ def getAccumulateCount(self):
         int
             The current value of AccumulateCount.
         """
-        return self._getSomethingSimple(self.at.getInt, Features.AccumulateCount)
+        return self._get_something_simple(self.at.getInt, Features.AccumulateCount)
 
     def setAccumulateCount(self, value):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/andorcamera.py:1064]: I think that `setAccumulateCount` is a candidate as well.

File: python/lsst/ts/genericcamera/driver/andorcamera.py [hunk 7]
```diff
@@ -1079,15 +1079,15 @@ def setAccumulateCount(self, value):
                 f"Value ({value}) for AccumulateCount must be "
                 f"between {minValue} and {maxValue}."
             )
-        self._setSomethingSimple(self.at.setInt, Features.AccumulateCount, value)
+        self._set_something_simple(self.at.setInt, Features.AccumulateCount, value)
 
     def cmdAcquisitionStart(self):
```
Comment (wvreeven) [inline python/lsst/ts/genericcamera/driver/andorcamera.py:1084]: I think that `cmdAcquisitionStart` is a candidate as well.

**Review by wvreeven — CHANGES_REQUESTED** [review]
Another great improvement. However, I found a few more candidates and will admit that I didn't scrutinize the code files. Please have another look and change all that are possible. Sorry to be a pain.

**Comment by pkubanek** [comment]
I removed hopefully last snakeCase artefacts, please have a look. Tests shall be passing.

**Comment by wvreeven** [comment]
Unfortunately one test case fails and flake8 isn't happy either. Please review the failures and fix the code accordingly.

**Review by wvreeven — APPROVED** [review]
Loooks MUCH better than before. The members of the `ASICameraInfo` class (in zwocamera.py) should be renamed as well. There still are several `_assertConnected` methonds in the liveview module. There are `startLiveView` and `stopLiveView` functions in the `EUI` class (in liveviewui.py). Otherwise fine.

**Pull request merged**